Traceback (most recent call last):
  File "$FILE", line 16, in <module>
    module.compile()
  File "$FILE", line 13, in test
    h(q)
ValueError: Value with linear type `qubit` was already used

Previous use occurred in $FILE:12 as an argument to `measure`
